juniper_official/routing/collect-rib-table-routes.rule

```diff
@@ -0,0 +1,129 @@
+/*
+ * Collects RIB table statistics for plotting and ML
+ */
+ healthbot {
+    topic routing.rib {
+        rule collect-rib-table-routes {
```

Comment:
no triggers, why is this being collected?

Reply:
Added trigger